instrumentation/ruby_kafka/opentelemetry-instrumentation-ruby_kafka.gemspec

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/instrumentation/ruby_kafka/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-instrumentation-ruby_kafka'
+  spec.version     = OpenTelemetry::Instrumentation::RubyKafka::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'RubyKafka instrumentation for the OpenTelemetry framework'
+  spec.description = 'RubyKafka instrumentation for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = ::Dir.glob('lib/**/*.rb') +
+               ::Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 2.5.0'
+
+  spec.add_dependency 'activesupport'
```

Comment:
Does it make sense to make `activesupport` a dependency here or let the consumer of the gem decide?

Reply:
Good question. I think having it here breaks assumptions that `opentelemetry-instrumentation-all` only pulls in OTel instrumentation. I'd prefer to check for it dynamically and, only if RubyKafka is present, warn that we can't instrument without it.

Reply:
Removed as a dependency here https://github.com/open-telemetry/opentelemetry-ruby/pull/476/commits/fed881efed4410fff6a4f0dae4bcf44f981314a2

I think it makes sense to still apply patches regardless of active support being present, as we don't need it for that level of instrumentation.